otterdog/eclipse-daanse.jsonnet

```diff
@@ -3,15 +3,15 @@ local orgs = import 'vendor/otterdog-defaults/otterdog-defaults.libsonnet';
 orgs.newOrg('eclipse-daanse') {
   settings+: {
     billing_email: "webmaster@eclipse-foundation.org",
-    dependabot_alerts_enabled_for_new_repositories: false,
-    dependabot_security_updates_enabled_for_new_repositories: false,
-    dependency_graph_enabled_for_new_repositories: false,
-    description: "",
+    dependabot_alerts_enabled_for_new_repositories: true,
+    dependabot_security_updates_enabled_for_new_repositories: true,
+    dependency_graph_enabled_for_new_repositories: true,
```

Comment:
if you change existing boolean values in your config from false to true, you can also remove them completely.
Your configuration basically captures the difference to the default configuration, thus if you remove this setting altogether, the value from the default configuration is getting applied.

There is also a playground to see the effects of the changes right away: https://eclipse-daanse.github.io/.eclipsefdn/playground/

Reply:
but this is not mandatory, you can also just leave the settings in your config and set it to true, just wanted to let you know.